src/components/ForecastSummary.js

```diff
@@ -1,45 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import WeatherIcon from "react-icons-weather";
-import "../styles/ForecastSummary.css";
-
-function ForecastSummary(props) {
-  const { date, description, icon, temperature, onSelect } = props;
```

Comment:
you can destructure in the brackets:
`function ForecastSummary({date, description, icon, temperature, onSelect) {`
and save yourself a line